src/libraries/ref.proj

```diff
@@ -1,8 +1,9 @@
 <Project Sdk="Microsoft.Build.Traversal">
-  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk" />
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.SharedFramework.Sdk" />
 
   <PropertyGroup>
     <TraversalGlobalProperties>BuildAllProjects=true</TraversalGlobalProperties>
+    <IsPackable>false</IsPackable>
```

Comment:
Why do we need this?

Reply:
We need this because the new sharedfx tooling sets IsPackable to True by default if it isn't set, and we have a target that this was causing to valid here: https://github.com/dotnet/runtime/blob/145467d84b6d1979d21a27731584cc8a6c0e35bd/Directory.Build.targets#L59-L63